spec/requests/payments/new_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'GET /payments/new', type: :request do
+  subject { response }
+
+  context 'without logged user' do
+    before do
+      stub_request(:post, 'https://api.iugu.com/v1/customers')
+        .to_return(status: 200, body: '', headers: {})
+      get '/payments/new'
+    end
+
+    it { is_expected.to have_http_status(:found) }
+  end
+
+  context 'with logged user' do
+    let!(:user) do
+      User.create!(
+        first_name: 'jane',
+        last_name: 'doe',
+        email: 'bar@baz.com',
+        password: '123456'
+      )
+    end
+
+    before do
+      headers = {
+        'Accept' => 'application/json',
+        'Authorization' => 'Basic MjU4NGFjNmQ5MGE4M2JlYzVmNjhmYmEzMzk0NmFiNWM6',
```

Comment:
CircleCI is failing here because this authorization string comes from the ENV variable. What should I do about this? Stub the ENV? @danielweinmann

Reply:
For these cases, there are two approaches I usually take:

1. I can create a `.env.test` variable with the ENV vars needed for specs and commit it to Git.

2. But you can also use a regex in the headers for `webmock`, like `'Authorization' => /Basic .+/` and skip the need for the ENV var.

For this case, I would go with number 2 ;)

Reply:
Ok! But the problem is that, since CircleCI doesn't have an API token, the 'authorization' part doesn't appear in its tests, that's what fails is.
So I'll do both, add a 'foo' api token to .env.test and a regex in the header to accept anything :)